nextjs-wordles/styles/globals.css

```diff
@@ -37,13 +36,34 @@
 
 
 .keyboard_block {
-    width: 3.5rem;
-    height: 3.5rem;
+    width: 4.5rem;
+    height: 4.5rem;
     background-color: rgb(224, 223, 223);
     border: 1px rgb(217, 213, 213) solid;
     display: flex;
     font-size: 1.5rem;
     align-items: center;
     justify-content: center;
     border-radius: 0.3rem;
+    cursor: pointer;
+}
+
+#Enter,#Delete,
+#A,#S,#D,#F,#A,#G,#H,#J,#K,#L,
+#Ą,#Ć,#Ę,#Ł,#Ń,#Ó,#Ś,#Ź,#Ż {
+    flex-grow:1;
+}
+
```

Comment:
Może dałoby się to jakoś uprościć? To bardzo trudne do utrzymania rozwiązanie.

Reply:
Niestety nie znalazłem żadnego zamiennika jak  na razie